src/index.ts

```diff
@@ -27,6 +27,9 @@ const safeSemver = (version: string) => {
 	// fix major.minor.patch.rev to use rev as build metadata
 	return (
 		version
+			// Replace any underscores with plusses, as they should
+			// have the same semantics
+			.replace(/_(\w+)/, '+$1')
```

Comment:
How about something a bit more stricter like:
```suggestion
			.replace(/([0-9]+\.[0-9]+\.[0-9]+)_(\w+)/, '$1+$2')
```
?